clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/config/KubernetesConfigurationProperties.groovy

```diff
@@ -32,6 +32,7 @@ class KubernetesConfigurationProperties {
     String cluster
     String oAuthServiceAccount
     List<String> oAuthScopes
+    String configuration
```

Comment:
Not a big fan of this name - is this supposed to be the kind of moniker that's applied? If so, maybe call this "namingStrategy"? And have the default one for k8s v2 be "namingStrategy = annotations" or something.

Reply:
Also it should probably be something that every account in clouddriver supplies, same way "permissions" is a field everywhere.